tests/test-sources/plugins/statuslines/lightline.nix

```diff
@@ -113,7 +113,7 @@
       enable = true;
 
       settings = {
-        colorscheme = "gruvbox";
+        colorscheme = "one";
```

Comment:
Also need to update the settingsExample, as per 4d012f8eb23f177a1b8a326ce1148a8837c7b55d